scripts/autograph_travis_test_config.yaml

```diff
@@ -0,0 +1,126 @@
+server:
+    # This port should be perfectly free, the upstream default of 8000
+    # is used by django sometimes so let's not do that.
+    listen: "0.0.0.0:5500"
+    # cache 500k nonces to protect from authorization replay attacks
+    noncecachesize: 524288
```

Comment:
nit: this will use a (very) small amount for no reason, feel free to reduce to `10`

Reply:
fair point, only for testing anyway.